tools/run_tests/build_artifact_python.sh

```diff
@@ -39,15 +39,6 @@ export PIP=${PIP:-pip}
 export AUDITWHEEL=${AUDITWHEEL:-auditwheel}
 
 
-if [ "$SKIP_PIP_INSTALL" == "" ]
```

Comment:
Why was this removed? When I said 'optimization' earlier, I meant that it was an optimization to skip this step, not an optimization to install.

Unless we no longer break when we don't have these lines? I guess we'll see in the watched build.